src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Same situation colleague.
Maybe ,it will be better, when Class have it by own field.
In this case , for better code support - enum.
Will discuss it later, comrade.

Reply:
Agree, it's better to use enum here